modules/metrics_routing/README.md

```diff
@@ -47,7 +50,7 @@ module "metric_router" {
             {
                 action = "send"
                 targets = [{
-                    id = module.observability_instances.metrics_router_targets["my-mr-target].id
+                    id = "c3af557f-fb0e-4476-85c3-0889e7fe7bc4"
```

Comment:
These seems odd. Is this a generic UID that everybody uses?

Reply:
Removed this.